### PR TITLE
Redesign UI

### PR DESCRIPTION
**Pull request by KaidenR**
Includes avatars for users, overlayed forms, and a file picker for repos

### CODE REVIEW

File: src/client/menu/repositories/components/add-repo-button/index.js [hunk 1]
```diff
@@ -1,59 +0,0 @@
-import React from 'react'
-import CSSModules from 'react-css-modules'
-
-import { func } from 'prop-types'
-
-import Button from '../../../components/button'
-
-import css from './index.css'
```
Comment (jimboslicethat) [inline src/client/menu/repositories/components/add-repo-button/index.js]: Not super important by any means, but the local imports could be grouped together.

File: src/package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "babel-register": "6.24.1",
     "babel-require": "1.0.1",
     "electron": "^1.7.9",
+    "md5": "^2.2.1",
```
Comment (jimboslicethat) [inline src/package.json]: We should be using `save-exact=true`. We should probably add that later, but for now I think just removing the carets should be good enough.

File: src/client/menu/components/button/index.js [hunk 3]
```diff
@@ -1,20 +1,37 @@
 import CSSModules from 'react-css-modules'
 import React from 'react'
 
-import { func, node } from 'prop-types'
+import { bool, func, node, oneOf, string } from 'prop-types'
 
 import css from './index.css'
 
+const TYPES = {
```
Comment (jimboslicethat) [inline src/client/menu/components/button/index.js]: Im not super opinionated on this, but wondering if there is a reason we aren't inlining these?

File: src/client/menu/index.js [hunk 4]
```diff
@@ -73,14 +74,37 @@ export default class Menu extends React.Component {
...
-  handleEditUser = async user => {
-    const users = userService.update(user)
-    this.setState({ users })
+  handleUserFormSubmit = () => {
+    this.state.userEdits.id ? this.handleSubmitEditUser() : this.handleSubmitAddUser()
+  }
+  handleEditUser = user => {
+    this.setState({ userEdits: { ...user } })
+  }
+  handleUserEditsChange = user => {
```
Comment (jimboslicethat) [inline src/client/menu/index.js]: It looks like `handleUserEditsChange` and `handleEditUser` are doing the same thing from what I can tell. I'd vote for just using `handleEditUser` and getting rid of the spread.

File: src/client/menu/index.js [hunk 5]
```diff
@@ -73,14 +74,37 @@ export default class Menu extends React.Component {
...
+    this.setState({ userEdits: user })
+  }
+  handleCancelEditUser = () => {
```
Comment (jimboslicethat) [inline src/client/menu/index.js]: Looks like this is the cancel for both add & edit, so maybe just `handleCancel`?

File: src/client/menu/repositories/components/add-repo-button/index.js (hunk 1 above)
Comment (KaidenR) [reply src/client/menu/repositories/components/add-repo-button/index.js]: This file was deleted =P

File: src/package.json (hunk 2 above)
Comment (KaidenR) [reply src/package.json]: Looks like we do have save-exact=true in the .npmrc. Not sure how these got in there. Good catch.

**Pull request merged**